File: Codeforces/Codeforces Round 878 (Div. 3)/A.py
# Chipher Shipher
# https://codeforces.com/contest/1840/problem/A
t = int(input())

for i in range(t):
    n = int(input())
    a = input() +";"
    a_x = a[0]
    res = ""
    i = 1
    while i < len(a)-1:
        if a_x == a[i]:
            res+=a_x
            a_x = a[i+1]
            i+=1
        i += 1

    print(res)

File: Codeforces/Codeforces Round 878 (Div. 3)/A.cpp
// Chipher Shipher
// https://codeforces.com/contest/1840/problem/A

#include "iostream"

using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--){
        int n;
        cin >> n;
        string ans, s;
        cin >> s;
        for (int i= 0; i < n; i++) {
            ans.push_back(s[i]);
            int tmp = i + 1;
            while (tmp < n - 1 and s[tmp] != s[i]) {
                tmp++;
            }
            i = tmp;
        }
        cout << ans << endl;
    }
    return 0;
}
